### DISCUSSION TITLE
Testing OpenVINO Whisper Transcription with Audacity 3.4.2

### DISCUSSION THREAD

**Discussion by petersampsonaudacity** [intel/openvino-plugins-ai-audacity#75] [Show and tell]
Testing the Whisper Transcription analyzer - 

I don't have any covert recordings with whispering so I tried Chris de Burgh's Lady in Red where he whispers "I love you" at the end.

Not only did it transcribe the whispered bit - it transcribed the whole lyrics of the song - mighty impressive.
Perhaps it needs a threshold setting to exclude non-whispered words?

![image](https://github.com/intel/openvino-plugins-ai-audacity/assets/50205984/23ab70e9-4232-4390-87c0-184b24437c7e)

**Comment by petersampsonaudacity** [comment]
It didn't fare so well with a track by a folk singer friend of mine who sings in Lancashire dialect - but I suspected it would struggle with that. 😉

But having said that it takes a fair stab at it, not easy to do with Michelle's dialect singing.

![image](https://github.com/intel/openvino-plugins-ai-audacity/assets/50205984/63a40387-f9af-4a0c-9c48-74249439f0c1)
The part outlined in Orange says (translated into normal English):
>If Pendle Hill were made of gold

Michelle's dialect sing made of gold as "med o gowd" or similar - no worries as her accompanist who is a UK southerner can't make that dialect pronunciation (nor me either) she winds us up about it.

**Comment by petersampsonaudacity** [comment]
@RyanMetcalfeInt8  replied (on Discord)

>And just to clarify, the AI model that we use is called 'whisper' (https://openai.com/research/whisper) -- but it's not really meant for *only* whispered speech.. so yeah, kind of confusing. It's intended for any kind of speech transcription, even if it's yelled 😉

**Comment by RyanMetcalfeInt8** [comment]
@petersampsonaudacity  -- Thank you for the feedback!

Tagged this also with 'enhancement', as there are bigger whisper models (small, large, etc.) that could technically add support for, which might give more accurate transcriptions / translations (at the cost of longer processing times).

**Comment by joedale100** [comment]
As OpenVINO Whisper Transcription is installed locally, does that mean it doesn't require sharing the audio files with third-party services, raising potential privacy and security concerns, particularly when handling sensitive or confidential material? I'm wondering if schools could use the plug-in safely and be compliant with the data handling policy. Could you clarify?

**Comment by RyanMetcalfeInt8** [comment]
Hi @joedale100 -- Yes, all of the AI models used by this plugin are installed and run locally. There are no third-party (cloud) services needed or used.. you could disconnect from the network and the plugins would work just fine.

**Comment by petersampsonaudacity** [comment]
@RyanMetcalfeInt8 

Hi Ryan,

I tested the transcription of track1 of Sgt Pepper, wich is a vocal track, but it found no words at all to transcribe:
![image](https://github.com/intel/openvino-plugins-ai-audacity/assets/50205984/85cead22-6dec-488e-990e-02c85d3747d3)

This of course tallies with the result I found earlier when testing noise suppression on #5

**Comment by RyanMetcalfeInt8** [comment]
Hi @petersampsonaudacity, 

> I tested the transcription of track1 of Sgt Pepper, wich is a vocal track, but it found no words at all to transcribe:

I can try this track on my end but these results may be expected -- I've had mixed results trying to transcribe vocals from a track that includes background instrumentals. You might have better luck by first applying music separation, and then applying transcription to the resulting vocal track.

Thanks!
Ryan
